components/lib/inputnumber/InputNumber.js

```diff
@@ -448,27 +457,31 @@ export const InputNumber = React.memo(
                 default:
                     break;
             }
-
-            if (props.onKeyDown) {
-                props.onKeyDown(event);
-            }
         };
 
-        const onInputKeyPress = (event) => {
+        const onInputKeyUp = (event) => {
             if (props.disabled || props.readOnly) {
                 return;
             }
 
-            const code = event.which || event.keyCode;
```

Comment:
This definition is missing and is the cause of the error.

Reply:
Fixed!